### ISSUE TITLE
libpagure has changed 

### ISSUE THREAD

**Issue by jscotka** [packit-service/ogr#64]
libpagure==0.20 changed `headers` handling, so that
code like:
https://github.com/packit-service/ogr/blob/master/ogr/services/our_pagure.py#L172
has to be adapted, currenly I've added dependency on lower version of libpagure to 
https://github.com/packit-service/ogr/pull/63/files#diff-380c6a8ebbbce17d55d50ef17d3cf906R44
to avoid this issue.

**Comment by TomasTomecek** [comment]
but sadly we can't pick libpagure version in Fedora; Honzo, this blocks only the mocking functionality, right?

**Comment by jscotka** [comment]
It is not about blocking mocking functionality, but  about using attributes and functions of libpagure and causes current troubles for testing via tox; because tox downloaded the latest version instead of using system library, what is fine to know env.
 
In fedora, right now there is good version, but the question is if eg in fedora rawhide isn't this version.

So that we had to add there some check for libpagure version. Trouble is with implementation of method ``get_raw_request`` in ``ourPagure``, what uses knowledge of attributes of ``Pagure`` class, what was changed.
